guides/common/modules/proc_distribution-of-archived-content-view-versions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Distribution_of_Archived_Content_View_Versions_{context}"]
+= Distribution of Archived Content View Versions
+
+If you have a Content View version that is not associated with any lifecycle environments, you can set the default view of that Content View version to be viewed.
```

Comment:
This line is a bit confusing. "Distribute archived content view versions" just allows non-promoted ("archived") content view version repositories to be exposed for consumption. It's useful for checking what content is in these content view versions during debugging. The whole feature in fact is mostly to help folks with debugging.

Reply:
Maybe change the opening to:
`The setting **Distribute archived content view versions** allows the hosting of archived Content View version repositories to be exposed for consumption. 
This is useful while debugging to see what content is present in your Content View versions.`

Reply:
"The setting **Distribute archived content view versions** allows archived Content View version repositories to be exposed for consumption."

or perhaps something like

"The setting **Distribute archived content view versions** enables hosting/exposure of non-promoted Content View version repositories in the Satellite content web application along with other repositories."